Header header
geometry_msgs/Transform[4] transform
uint8 n_tracker

uint8 LEFT = 0
uint8 RIGHT = 1
 
